auditlog/management/commands/auditlogmigratejson.py

```diff
@@ -0,0 +1,108 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.paginator import Paginator
+
+from auditlog.models import LogEntry
+
+
+class Command(BaseCommand):
+    help = "Migrates changes from changes_text to json changes."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-d",
+            "--database",
+            default=None,
+            help="If provided, we will use native db operations.",
+            dest="db",
+            type=str,
+            choices=["postgres", "mysql", "oracle", "sqllite3"],
```

Comment:
I think we don't support `sqllite3`

Reply:
Also it's `sqlite3`, single `l`.

Reply:
Oh. I saw it included in https://github.com/jazzband/django-auditlog/pull/407#issuecomment-1205656698 and I thought it was supported.